diferencia.txt

```diff
@@ -0,0 +1,6 @@
+Este es un archivo que sólo se debe encontrar en el nuevo branch.
+Me pregunto cómo agregarlo al repositorio. Seguro es con git add diferencia.txt
+Veremos.
```

Comment:
Es correcto, Alonso. Esto es inútil, pero tampoco estorba, así que lo podemos dejar.
:wink: